users/src/main/scala/users/services/usermanagement/domain.scala

```diff
@@ -0,0 +1,13 @@
+package users.services.usermanagement
+
+import scala.util.control.NoStackTrace
+
+sealed trait Error extends Throwable with NoStackTrace
+object Error {
+  final case object AlreadyExists extends Error
+  final case object NotFound extends Error
+  final case object Active extends Error
+  final case object Deleted extends Error
+  final case object Blocked extends Error
+  final case class System(underlying: Throwable) extends Error
```

Comment:
Unused. Can this be returned somewhere? Think it'd be interesting to see how raw exceptions are handled by the API.

Reply:
I know it's unused, and putting it in is not something I can do in the next 10 minutes. I'll leave it in as a case, so they have to handle it anyway (even if it doesn't pop up)